options/options.go

```diff
@@ -109,12 +118,17 @@ func NewTerragruntOptions(terragruntConfigPath string) *TerragruntOptions {
 		RunTerragrunt: func(terragruntOptions *TerragruntOptions) error {
 			return errors.WithStackTrace(RunTerragruntCommandNotSet)
 		},
-	}
+	}, nil
 }
 
 // Create a new TerragruntOptions object with reasonable defaults for test usage
 func NewTerragruntOptionsForTest(terragruntConfigPath string) *TerragruntOptions {
-	opts := NewTerragruntOptions(terragruntConfigPath)
+	logger := util.CreateLogger("")
+
+	opts, err := NewTerragruntOptions(terragruntConfigPath)
+	if err != nil {
+		logger.Panicf("error: %v\n", err)
```

Comment:
This results in a `panic`. I think this method would need to return an error obj too :-\